src/collapse/collapse.js

```diff
@@ -26,6 +29,8 @@ angular.module('ui.bootstrap.collapse', [])
             // prevents the animation from jumping to collapsed state
             .removeClass('collapse')
             .addClass('collapsing');
+          element.attr('aria-expanded', false);
```

Comment:
You can chain on the other calls to `element` right after `.addClass(...)` instead of calling it again